Sources/FodyRunner.UnityIntegration/InspectorEditor.cs

```diff
@@ -282,30 +282,50 @@ protected virtual void ApplyChangeHandlersToProperty(Object targetObject, Serial
         protected virtual void ProcessObjectProperties(SerializedProperty property)
         {
             string undoRedoWarningPropertyPath = UndoRedoWarningPropertyPath;
-
             do
             {
-                string propertyPath = property.propertyPath;
-                Object targetObject = property.serializedObject.targetObject;
+                ProcessObjectProperty(property, undoRedoWarningPropertyPath);
```

Comment:
Do we need to pass a local variable that is a copy of `UndoRedoWarningPropertyPath` around like this? I guess it's a performance optimization as that's a property that does a bunch of logic in its getter.

Reply:
Yeah, I was just copying the original string that was being set at the start of the do/while loop through the properties.

I figured it was being done once as to make it more efficient.